superbit_lensing/galsim/mock_superBIT_data.py

```diff
@@ -352,8 +358,9 @@ def make_cluster_galaxy(ud, wcs,affine, centerpix, cluster_cat, optics, sbparams
     cluster_galaxy_truth.g1=g1; cluster_galaxy_truth.g2=g2
```

Comment:
You rightly pointed out in a previous PR that we should adopt a consistent style for spaces between commas, equals signs, and so on. At some point (not now) we should decide on a standard usage and then enforce it in the codebase.

Reply:
I have slowly adapted lots of the refactored code to be this way, but sometimes it hasn't been worth it (like in the `medsmaker` module). I've just thought of this as a long-term project :)

But yes, to work towards that we should have a few expectations for all new code